tests/MongoDB.Driver.Legacy.Tests/Operations/BulkWriteOperationTests.cs

```diff
@@ -202,28 +201,48 @@ public void TestExecuteTwice(bool ordered)
             Assert.Throws<InvalidOperationException>(() => bulk.Execute());
         }
 
+        public static MongoClientSettings GetClientSettings()
+        {
+            var connectionString = CoreTestConfiguration.ConnectionString.ToString();
+            var clientSettings = MongoClientSettings.FromUrl(new MongoUrl(connectionString));
+
+            var serverSelectionTimeoutString = Environment.GetEnvironmentVariable("MONGO_SERVER_SELECTION_TIMEOUT_MS");
+            if (serverSelectionTimeoutString == null)
+            {
+                serverSelectionTimeoutString = "30000";
+            }
+
+            clientSettings.ServerSelectionTimeout = TimeSpan.FromMilliseconds(int.Parse(serverSelectionTimeoutString));
+            clientSettings.ClusterConfigurator = cb => CoreTestConfiguration.ConfigureLogging(cb);
+
+            return clientSettings;
+        }
+
         [Theory]
         [InlineData(false, 0)]
         [InlineData(false, 1)]
         [InlineData(true, 0)]
         [InlineData(true, 1)]
         public void TestExecuteWithExplicitWriteConcern(bool ordered, int w)
         {
+            var server = GetServerFromDisposableClient();
```

Comment:
Do this instead:

    var server = LegacyTestConfiguration.GetServer(retryWrites: false);

Reply:
Done